docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -91,11 +91,11 @@ A request sent from the query-tee to a backend includes HTTP basic authenticatio
 
 ### Backend selection
 
-The query-tee supports always sending requests to all backends, or sending only a proportion of requests to all backends, with the remainder sent only to the preferred backend.
-This is configured with `-proxy.secondary-backends-request-proportion`.
+You can use the query-tee to either send requests to all backends, or to send a proportion of requests to all backends and the remaining requests to only the preferred backend.
+You can configure this with `-proxy.secondary-backends-request-proportion`.
```

Comment:
I like specifying what `-proxy.secondary-backends-request-proportion` is a setting. For example, "You can configure this with the `-proxy.secondary-backends-request-proportion` setting. Otherwise, looks greats!

Reply:
Makes sense, I've updated it to be clearer. I've used the term "CLI flag" rather than "setting" as that's more specific and appropriate for this case.